frontend/components/modal/ModalSharePage.vue

```diff
@@ -196,6 +200,7 @@ import type { Group } from "~/types/group";
 import type { Organization } from "~/types/organization";
 import type { Resource } from "~/types/resource";
 import type { User } from "~/types/user";
+import IconSignal from "../icon/IconSignal.vue";
```

Comment:
In Nuxt we actually don't need to import the components, so this can be removed (regardless of us not using a component anyway) 😊